pkg/validate/validate_values.go

```diff
@@ -37,8 +37,8 @@ func CheckValues(root map[string]interface{}) util.Errors {
 	if err != nil {
 		return util.Errors{err}
 	}
-	val := &v1alpha2.Values{}
-	if err := yaml.Unmarshal(vs, val); err != nil {
+	val := &v1alpha22.Values{}
```

Comment:
I assume this is temporary - ultimately we'll want to move this into the same v1alpha2 package.

Reply:
not sure whether we want to do so since there might be some conflict between each other if we put them into same package. Also essentially they belong to two groups of APIs, the values one is for validation purpose only. So I think it might better to keep it in different package.